results/sentence-transformers__paraphrase-multilingual-MiniLM-L12-v2/IN22ConvBitextMining.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
The task is nearly impossible, could you try with `intfloat/multilingual-e5-small` please?

Reply:
I have added the results between English and a few low-resource pairs which seem to be reasonable. However, I agree that the performance between low resource pairs seem to be very low.

Reply:
Thanks, you're just missing the points and we'l be good for merging 🙂

Reply:
I have added the points for dataset addition and PR review. Thanks!